### PR TITLE
adding talus terms

### PR DESCRIPTION
**Pull request by megbalk**
adding: 
trochlea tali
trochlear groove of talus
ridge of lateral trochlea of talus
ridge of medial trochlea of talus
condyle of talus
medial condyle of talus
lateral condyle of talus
sulcus tali

Issues #1880, #1881

### CODE REVIEW

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by matentzn** [comment]
@megbalk QC is failing, please fix before we proceed to merge.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224701,6 +224701,85 @@ intersection_of: connects UBERON:7500034 ! distal-most point of trochlea of hume
...
+intersection_of: part_of UBERON:0002395 ! talus
+property_value: http://purl.org/dc/elements/1.1/date 2021-05-28T00:59:42Z xsd:dateTime
+property_value: term:editor http://orcid.org/0000-0003-2699-3066
+
+[Term]
+id: UBERON:7500066
+name: medial condyle of talus
+def: "The medial condyle of talus is an articular prominence of bone on medial side of the trochlea of a talus, which engages with the tibia." [ORCID: 0000-0003-2699-3066]
+synonym: "medial condyle of astragalus" EXACT []
+is_a: UBERON:7500065 ! condyle of talus
```
Comment (dosumis) [inline src/ontology/uberon-edit.obo:224742]: you can let the reasoner infer this from 
is_a: condyle
relationship: part_of talus

Dynamic is generally better.

**Review by dosumis — CHANGES_REQUESTED** [review]
See comments

**Comment by dosumis** [comment]
QC fail appears to be lack of reference on definition:

image

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (megbalk) [reply src/ontology/uberon-edit.obo:224742]: done.

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224701,6 +224701,85 @@ intersection_of: connects UBERON:7500034 ! distal-most point of trochlea of hume
...
 created_by: https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500063
+name: trochlea of talus
+def: "The articular surface of a talus that interacts with a tibia. It has a deep depression that forms the trochlear groove and is bounded by the medial and lateral condyles." [https://en.wikipedia.org/wiki/Talus_bone]
+synonym: "trochlea for tibia" EXACT []
+synonym: "trochlea of astragalus" EXACT []
+synonym: "trochlea tali" EXACT []
+is_a: UBERON:0004120 ! mesoderm-derived structure
```
[the remark this answers is not in the document]
Comment (megbalk) [reply src/ontology/uberon-edit.obo:224711]: Added 'surface of bone'. Should I instead add a parent class 'articular surface for the tibia on the astragalus'?

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -224701,6 +224701,85 @@ intersection_of: connects UBERON:7500034 ! distal-most point of trochlea of hume
...
+is_a: UBERON:7500066 ! medial condyle of talus
+property_value: http://purl.org/dc/elements/1.1/date 2021-05-28T01:03:10Z xsd:dateTime
+property_value: term:editor http://orcid.org/0000-0003-2699-3066
+
+[Term]
+id: UBERON:7500069
+name: ridge of lateral condyle of talus
+def: "The crest of the lateral condyle of a talus." [ORCID: 0000-0003-2699-3066]
+synonym: "ridge of lateral condyle of astragalus" EXACT []
+is_a: UBERON:7500067 ! lateral condyle of talus
```
[the remark this answers is not in the document]
Comment (megbalk) [reply src/ontology/uberon-edit.obo:224769]: done. did the same for ridge of medial condyle of talus.

File: src/ontology/uberon-edit.obo [hunk 4]
```diff
@@ -224701,6 +224701,85 @@ intersection_of: connects UBERON:7500034 ! distal-most point of trochlea of hume
...
+property_value: http://purl.org/dc/elements/1.1/date 2021-05-28T01:03:51Z xsd:dateTime
+property_value: term:editor http://orcid.org/0000-0003-2699-3066
+
+[Term]
+id: UBERON:7500070
+name: sulcus tali
+def: "A depression that runs between the two facets of a talus that interact with a calcaneum." [https://en.wikipedia.org/wiki/Talus_bone]
+synonym: "sulcus for flexor hallucis longus tendon" EXACT []
+intersection_of: UBERON:0000093 ! sulcus
+intersection_of: part_of UBERON:0002395 ! talus
```
[the remark this answers is not in the document]
Comment (megbalk) [reply src/ontology/uberon-edit.obo:224779]: how is this different from what I have? should these be subclass relationships?

[the remark this answers is not in the document]
Comment (megbalk) [reply src/ontology/uberon-edit.obo:224779]: done.

File: src/ontology/uberon-edit.obo [hunk 5]
```diff
@@ -224701,6 +224701,85 @@ intersection_of: connects UBERON:7500034 ! distal-most point of trochlea of hume
...
+property_value: http://purl.org/dc/elements/1.1/date 2021-05-28T00:56:19Z xsd:dateTime
+property_value: term:editor http://orcid.org/0000-0003-2699-3066
+
+[Term]
+id: UBERON:7500064
+name: trochlear groove of talus
+def: "The furrow between the lateral and medial condyles of the trochlea of talus." [https://en.wikipedia.org/wiki/Talus_bone]
+synonym: "trochlear groove of astragalus" EXACT []
+is_a: UBERON:0000064 ! organ part
+is_a: UBERON:0004120 ! mesoderm-derived structure
```
[the remark this answers is not in the document]
Comment (megbalk) [reply src/ontology/uberon-edit.obo:224722]: done.

File: src/ontology/uberon-edit.obo (hunk 2 above)
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/uberon-edit.obo:224711]: That would make sense - how well conserved is the function?

Given the limited curation of articular surfaces into Uberon, we might want to discuss this as part of a more general attempt to patternise.

[the remark this answers is not in the document]
Comment (megbalk) [reply src/ontology/uberon-edit.obo:224711]: I think pretty well conserved. I'll add it.

[the remark this answers is not in the document]
Comment (megbalk) [reply src/ontology/uberon-edit.obo:224711]: done.

File: src/ontology/uberon-edit.obo [hunk 6]
```diff
@@ -216,6 +216,15 @@ treat-xrefs-as-reverse-genus-differentia: XAO part_of NCBITaxon:8353
 treat-xrefs-as-reverse-genus-differentia: ZFA part_of NCBITaxon:7954
 treat-xrefs-as-reverse-genus-differentia: ZFS part_of NCBITaxon:7954
 
+[Term]
+id: UBEORN:7500081
+name: articular surface for the tibia on the talus
+def: "Conact surface between the tibia and the talus located on the talus." [https://en.wikipedia.org/wiki/Talus_bone]
+is_a: UBERON:4100003 ! articular surface
```
Comment (dosumis) [inline src/ontology/uberon-edit.obo:223]: Typo. Conact -> Contact

Comment (megbalk) [reply src/ontology/uberon-edit.obo:223]: fixed!

**Review by dosumis — APPROVED** [review]

**Pull request merged**